src/sentry/api/endpoints/project_details.py

```diff
@@ -274,6 +277,33 @@ def put(self, request, project):
                     'sentry:blacklisted_ips',
                     clean_newline_inputs(options['filters:blacklisted_ips'])
                 )
+            if 'filters:{}'.format(FilterTypes.RELEASES) in options:
+                if features.has('projects:additional-data-filters', project, actor=request.user):
+                    project.update_option(
+                        'sentry:{}'.format(FilterTypes.RELEASES),
+                        clean_newline_inputs(options['filters:{}'.format(FilterTypes.RELEASES)])
+                    )
+                else:
+                    return Response(
+                        {
+                            'detail': ['You do not have that feature enabled']
+                        }, status=400
+                    )
```

Comment:
I agree. Open to other solutions, but it also brings up the question of why we're gating these specifically.